elasticdl/python/master/servicer.py

```diff
@@ -91,7 +92,7 @@ def _init_model_from_var_list(self, var_list):
     def _init_model_from_tensor_list(self, tensor_list):
```

Comment:
tensor_list is a tensor_pb_list? We may need rename it to avoid the confusion.
Because tensor_list and tensor_pb_list co-exist in our code base.

Reply:
Good point. I did not notice this function. Done.

By the way, I am using three names now, `tf_tensor` for TensorFlow's tensor, `tensor_pb` for tensor protocol buffer, and `tensor` for ElasticDL's tensor.